src/theory/strings/theory_strings_rewriter.h

```diff
@@ -109,6 +109,12 @@ class TheoryStringsRewriter {
   * Returns the rewritten form of node.
   */
   static Node rewritePrefixSuffix(Node node);
+  /** rewrite str.code
+   * This is the entry point for post-rewriting terms n of the form
+   *   str.code( t )
+   * Returns the rewritten form of node.
```

Comment:
I think this would be too verbose for documentation in the h file. The rewriters for each of the string functions are very complex and subject to change.

Reply:
I agree that it is not necessary to document this in the header but to keep that documentation in the body of the function.